src/main/cpp/subsystems/SwerveDrive.cpp

```diff
@@ -112,34 +65,21 @@ bool SwerveDrive::Init() {
 
     // setpoint
     at_setpoint_ = false;
-    
 
     // Reset everything
     OKC_CALL(ResetDriveEncoders());
     OKC_CALL(ResetGyro());
     return true;
 }
 
-bool SwerveDrive::UpdateModules() {
-    // update modules
-    OKC_CALL(left_front_module_->Update(this->interface_->left_front_drive_motor_enc, this->interface_->left_front_steer_motor_enc, this->interface_->left_front_drive_enc_vel, this->interface_->left_front_steer_enc_vel));
-    OKC_CALL(left_back_module_->Update(this->interface_->left_back_drive_motor_enc, this->interface_->left_back_steer_motor_enc, this->interface_->left_back_drive_enc_vel, this->interface_->left_back_steer_enc_vel));
-    OKC_CALL(right_front_module_->Update(this->interface_->right_front_drive_motor_enc, this->interface_->right_front_steer_motor_enc, this->interface_->right_front_drive_enc_vel, this->interface_->right_front_steer_enc_vel));
-    OKC_CALL(right_back_module_->Update(this->interface_->right_back_drive_motor_enc, this->interface_->right_back_steer_motor_enc, this->interface_->right_back_drive_enc_vel, this->interface_->right_back_steer_enc_vel));
-
-    // update module positions
-    OKC_CALL(left_front_module_->GetSwerveModulePosition(left_front_pos_.get()));
-    OKC_CALL(left_back_module_->GetSwerveModulePosition(left_back_pos_.get()));
```

Comment:
Do we not need to update the module positions using this anymore?

Reply:
. . . we still need to update module positions, yes. I thought I reinstated this, but I guess not. . .

Reply:
oh, yeah, I just call them in Periodic() instead of having a separate method. I can have a separate method if we need it though

Reply:
I'm a fan of making separate functions for every distinct action the robot can take because it's way easier to debug than sifting through a pile of code in Periodic. But this is fine for now since it's more a matter of technical debt